package.json

```
@@ -67,7 +67,7 @@
     "@biomejs/biome": "^1.8.3",
     "@changesets/cli": "^2.27.6",
     "@fleek-platform/errors": "^2.5.1",
-    "@fleek-platform/sdk": "^3.4.0",
+    "@fleek-platform/sdk": "workspace:*",
```

I'll pick it from now. So personal note, on sdk release

```suggestion
    "@fleek-platform/sdk": "^3.5.0",
```